tests/product/test_server_uninstall.py

```diff
@@ -108,54 +78,8 @@ def test_uninstall_lost_host(self):
         cmd_output = self.run_prestoadmin('server uninstall',
                                           raise_error=False)
         self.assertRegexpMatches(cmd_output, expected)
-        process_per_active_host = []
-        for host, pid in process_per_host:
-            if host not in down_node:
-                process_per_active_host.append((host, pid))
-        self.assert_stopped(process_per_active_host)
```

Comment:
I assume we don't want to test this in a product test because it's probably already covered in a unit test, i.e. that uninstall calls stop and therefore asserting that everything is stopped is unnecessarily testing the stop action?